book/src/proptest-derive/modifiers.md

```diff
@@ -0,0 +1,370 @@
+# Modifier Reference
+
+All modifiers interpreted by `#[derive(Arbitrary)]` are of the form
+`#[proptest(..)]`, where the content between the parentheses follows the normal
+Rust attribute syntax.
+
+Each modifier within the parentheses is independent, in that putting two
+modifiers in the same attribute is equivalent to having two `#[proptest(..)]`
+attributes with one modifier each.
+
+For brevity, modifiers are sometimes referenced by name alone; e.g., "the
+`weight` modifier" refers to `#[proptest(weight = nn)]` and not some
+freestanding `#[weight]` attribute.
```

Comment:
Hmm... I think `weight = W` works, but a reader might now have read this bit and is jumping around and so `#[weight]` could be confusing...